### PR TITLE
feat: footer nav

### LINKED ISSUE #4: Implement Footer Navigation
Desktop:
Screenshot 2023-02-07 at 11 05 14

Mobile:
Screenshot 2023-02-07 at 11 05 00

### PR DESCRIPTION
**Pull request by dzehnder**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #4 

Test URLs:
- Before: https://main--choice--hlxsites.hlx.page/en-us/explore/
- After: https://footer-nav--choice--hlxsites.hlx.page/en-us/explore/

### CODE REVIEW

**Comment by solaris007** [comment]
we do have CLS from previously zero

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -9,23 +9,56 @@ footer .block {
...
 footer .footer p {
   margin: 0;
 }
 
-.columns > div {
-  display: flex;
-  flex-direction: column;
+footer .columns p {
+  max-width: 240px;
+  color: rgb(0 0 0 / 55%);
```
Comment (solaris007) [inline blocks/footer/footer.css:24]: should we standardize on using hex or rgb? could also be expressed as `0000008C`

File: styles/styles.css [hunk 2]
```diff
@@ -15,33 +15,46 @@
...
   --text-color: #000;
   --footer-border-color: #e4e4e4;
   --header-border-color: #ffc72c;
-  --header-nav-top-color: #212121;
+  --font-black: #212121;
   --header-language-color: rgb(0 0 0 / 55%);
   --header-language-hover-color: rgb(0 0 0 / 80%);
   --header-nav-language-select: #979797;
+  --card-background-color: #fafafa;
+  --card-box-shardow: 0 1px 5px 0 rgb(119 112 91 / 20%), 0 3px 4px 0 rgb(119 112 91 / 12%), 0 2px 4px 0 rgb(119 112 91 / 14%);
```
Comment (solaris007) [inline styles/styles.css:57]: hex vs rgb

**Review by solaris007 — APPROVED** [review]

File: blocks/footer/footer.css (hunk 1 above)
Comment (dzehnder) [reply blocks/footer/footer.css:24]: IMO its easier to read in rgb when there is an alpha channel. But generally I have no strong opinion on it

**Comment by solaris007** [comment]
Screenshot 2023-02-10 at 16 36 09

- margin above social links
- join box probably too narrow (see text wrap)

**Comment by dzehnder** [comment]
Remaining layout shift comes from sample page:
Screenshot 2023-02-10 at 16 47 32

**Comment by solaris007** [comment]
@dzehnder we should use the explore page for LHS, i updated the GH PR template and your PR
```
Test URLs:
- Before: https://main--choice--hlxsites.hlx.page/en-us/explore/
- After: https://footer-nav--choice--hlxsites.hlx.page/en-us/explore/
```

**Comment by solaris007** [comment]
did you merge main with the hero styles? it mitigates some of the CLS

**Comment by dzehnder** [comment]
Even after merging, seems like still the same CLS issue:
Screenshot 2023-02-10 at 17 06 55

**Review by solaris007 — APPROVED** [review]

**Pull request merged**